### PR TITLE
CI set-up frontend container image continuous creation

### PR DESCRIPTION
**Pull request by carlosthe19916**

### CODE REVIEW

File: frontend/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 FROM registry.access.redhat.com/ubi9/nodejs-20:latest as builder
```
Comment (bobmcwhirter) [inline frontend/Dockerfile:2]: Should we name this `Containerfile`?

**Review by bobmcwhirter — CHANGES_REQUESTED** [review]

File: frontend/Dockerfile (hunk 1 above)
Comment (helio-frota) [reply frontend/Dockerfile:2]: Yes https://github.com/containers/buildah/discussions/3170#discussion-3330099

Comment (helio-frota) [reply frontend/Dockerfile:2]: On the previous PR I missed a tag 
 io.openshift.tags="operator,trustification,trustify,ui,nodejs18" \
 
 should be:
 
 io.openshift.tags="operator,trustification,trustify,ui,nodejs20" \

File: frontend/Containerfile [hunk 2]
```diff
@@ -15,18 +15,18 @@ RUN microdnf -y install tar procps-ng && microdnf clean all
...
-      io.k8s.display-name="trustification-ui" \
-      io.k8s.description="Trustification - User Interface" \
+      summary="Trustify - User Interface" \
+      url="https://ghcr.io/trustification/trustify-ui" \
+      usage="podman run -p 80 -v trustification/trustify-ui:latest" \
+      io.k8s.display-name="trustify-ui" \
+      io.k8s.description="Trustify - User Interface" \
       io.openshift.expose-services="80:http" \
-      io.openshift.tags="operator,trustification,ui,nodejs18" \
+      io.openshift.tags="operator,trustification,trustify,ui,nodejs20" \
```
Comment (helio-frota) [inline frontend/Containerfile:29]: thanks Carlos :+1:

File: frontend/Dockerfile (hunk 1 above)
Comment (carlosthe19916) [reply frontend/Dockerfile]: > Should we name this `Containerfile`?

We can, and I've renamed it

> Should we name this `Containerfile`?

Done

**Pull request merged**